scripts/install.sh

```diff
@@ -8,8 +8,9 @@ read -p "Вы уверены, что хотите установить скри
 if [[ $ans == "y"* || $ans == "Y"* || $ans == "д"* || $ans == "Д"* ]]; then
 
     mkdir -p -v ~/$home_dir/KORYAVNIKS
-    cp config.conf ~/$home_dir
-    sudo cp $scripts_dir/run.sh /usr/local/bin/$home_dir
+    cp ../config.conf ~/$home_dir
+    sudo cp run.sh /usr/local/bin/$home_dir
     
     echo "Cкрипт gnu-koryavov установлен."
```

Comment:
Probably there is better to use either Russian or English in all output messages.